### ISSUE TITLE
WT pass-band filter for the Cls

### ISSUE THREAD

**Issue by dlanzieri** [LSSTDESC/DifferentiableHOS#22]
To be sure we were using the same scales for the PS and the HOS we have applied a pass-band filter to the maps before taking the power spectrum. We  have compute the WT of the input convergence map,  summed back the wavelet scales 4 and 5 and computed the PS of that. 
This corresponds to use the WT to isolate particular scales. 
In the following plot the dashed curve is the power spectrum described above, the orange curve is the ps from a traditional smoothed map, the red and green are the ps computed from the 4th and 5th WT scales.
![comp_ps (1)](https://user-images.githubusercontent.com/72620117/167711804-7ce859cb-3d15-4184-9db9-533752698e0f.png) .
So @EiffL , this could explain why the contours for the wt filtered maps are tighter.

**Comment by EiffL** [comment]
ok, yeah, that explains it :-) But also note that we can see on this plot that the wavelet transform in green goes to like ell=5000, 3000 is right where it peaks

**Comment by EiffL** [comment]
So, if we want to limit to ell=3000, we should use only the red scale... Either that, or we use the power spectrum up to ell=5000 or 6000 as well and then we can use 2 scales.

**Comment by dlanzieri** [comment]
> So, if we want to limit to ell=3000, we should use only the red scale... Either that, or we use the power spectrum up to ell=5000 or 6000 as well and then we can use 2 scales.

Maybe I can see  how the results are compared to KTNG with this new type of smoothing, to see how much we can trust of the ps up to these values. What do you think?

**Comment by EiffL** [comment]
Yeah, but I don't think it's going to make a difference. On the angular Cell residual plots the smoothing has no effect.
